src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  display: flex;
+  width: $width-calendar;
+  padding: 10px;
+  flex-wrap: wrap;
+  gap: $gap;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+
+  &__cell {
+    display: flex;
+    @include square($cell-size);
+    align-items: center;
+    justify-content: center;
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: 0.5s;
+  }
```

Comment:
You should specify transition properties directly
Default is all, but it could affect old browsers' performance